portfolio/src/main/webapp/scripts/templatemo-script.js

```diff
@@ -42,20 +42,12 @@
 })(jQuery);
 
 $(document).ready(function () {
-
-    // Check for video loading complete
-    var video = document.getElementById("tm-welcome-video");
-
-    video.onloadeddata = function() {
-        $('#tm-video-text-overlay').removeClass('d-none');
-        $('#tm-video-loader').addClass('d-none');
-
-        $('#rotate').rotaterator({
-            fadeSpeed: 1000,
-            pauseSpeed: 300
-        });
-    }
-
     // Update year in copyright text
     document.querySelector('.tm-current-year').textContent = new Date().getFullYear();
 });
+
+async function fetchJson() {
+    const response = await fetch('/data');
+    const json = await response.text();
+    document.querySelector('#comments').innerHTML = json;
```

Comment:
since the returned html is a list, should we loop through it? (adding <p> for example)